travis-scripts/setup-php-fpm.sh

```diff
@@ -19,6 +19,8 @@ echo "cgi.fix_pathinfo = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/con
 echo "always_populate_raw_post_data = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 echo "error_log = /var/log/php-fpm.log" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 echo "memory_limit = 2G" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+echo "opcache.revalidate_freq=7000" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```

Comment:
@sowbiba Why do you change these values ?

Reply:
I had an error from travis
https://stackoverflow.com/questions/50769286/zend-mm-heap-corrupted-with-standard-php-5-6-locating-the-cause
https://bugs.php.net/bug.php?id=69540

Reply:
Relative only for PHP 5.6 ?

Reply:
yes only for 5.6

Reply:
Don't need to add it. Rerun travis, and everything will be ok (sometimes :sweat_smile: )

Reply:
May be these settings should be conditional to PHP 5.6 ?

Reply:
Since we remove the 5.6 support in 1.7.7.0 and only related to tests, we should maybe ignore it wdyt?

Reply:
Rerun, I have passed my last PR like it.